tools/cldr-rdf/src/main/java/org/unicode/cldr/tool/GenerateLanguageContainment.java

```diff
@@ -418,68 +465,72 @@ private static void log(String string) {
 //        }
     }
 
-    private static String getBest(Collection<String> parents) {
-        for (String parent : parents) {
-            String code = entityToCode.get(parent);
-            if (code == null) continue;
-            Type type = Iso639Data.getType(code);
-            if (type != Type.Living) {
-                continue;
-            }
-            return parent;
-        }
-        // failed
-        return parents.iterator().next();
-    }
-
-    private static String getEntityName(String key) {
-        String code = entityToCode.get(key);
-        if (code != null) {
-            try {
-                String name = NAME.apply(code);
-                if (name != null) {
-                    return name;
-                }
-            } catch (Exception e) {
-            }
-        }
-        String name = entityToLabel.get(key);
-        if (name != null) {
-            return name;
-        }
-        int last = key.lastIndexOf('/');
-        return key.substring(last + 1, key.length() - 1);
-    }
-
-    private static Multimap<String, String> loadTsvPairs(Class<?> class1, String file,
-        Function<String, String> keyMapper, Function<String, String> valueMapper) {
-        String rel = FileUtilities.getRelativeFileName(class1, file);
-        System.out.println(rel);
+//    private static String getBest(Collection<String> parents) {
+//        for (String parent : parents) {
```

Comment:
better to delete than to comment out

Reply:
True, in this case it's used by other code that's also commented out at https://github.com/unicode-org/cldr/pull/900/files/89fe206ccd3bc8cfbd4b308148d21e4d190ed14f#diff-f5160be6c0e53da3045f35879f2bab04b51ce622ab6899c90062a6cc1bb3bf44L387

Reply:
Perhaps I could add a "TODO: only used by commented out code" here?